front/src/api/todos-api.ts

```diff
@@ -0,0 +1,52 @@
+import { Todo, TodosArr } from '../types';
+import { APIResponseType, axiosInstance } from './api';
+import { fromDTO, parseTodoDTO } from '../helpers';
+
+export const todosAPI = {
+  getAllTodos() {
+    return axiosInstance.get<APIResponseType<TodosArr>>('/todos')
+      .then(response => {
+        const todosArr = response.data.data.map(todo => {
```

Comment:
well, it's up to backend to decide what structure is. But I'd expect `response.data` to be `TodoDTO[]` already, without nesting into one more `data`

Reply:
@skyboyer oh, I agree that it looks really strange. But the thing is:
1) the first `data` prop creates axios. So if we want to see any data we should always check the prop `response.data`
![image](https://user-images.githubusercontent.com/23404373/190873466-f90c6c03-4f23-46f5-8559-81a4060ae52d.png)
2) the second data it is our data. Or we should not return `meta` prop anymore in the server's response?
![image](https://user-images.githubusercontent.com/23404373/190873560-a30f9e41-4ef7-4f83-b6f6-61def9ffdc62.png)